core/themes/basis/basis.info

```diff
@@ -0,0 +1,30 @@
+name = Basis
+description = The default front-end theme for Backdrop CMS, which can make a great base theme.
+type = theme
+backdrop = 1.x
+version = BACKDROP_VERSION
+
+stylesheets[all][] = css/base.css
+stylesheets[all][] = css/layout.css
+stylesheets[all][] = css/component/small-text-components.css
+stylesheets[all][] = css/component/l-header.css
```

Comment:
a theme shouldn't target a layout selector.  .l-header is for layout/positioning only (and layout/positioning CSS should be left to the layout selected). This is important for the separation of layout + design and we shouldn't be disobeying this rule in core if we expect all contrib + custom themes to do so.

 Can we ...
* change this file name to header.css 
* have the CSS target the 'header` element instead (or a different class added by basis)